.pre-commit-config.yaml

```diff
@@ -0,0 +1,9 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+-   repo: local
+    hooks:
+      - id: fmt
```

Comment:
sounds useful (although for me it's already muscle memory :D)

Reply:
I think it's good to avoid the CI running for nothing; CI often fails because of a formatting issue.